renovate.json

```diff
@@ -25,13 +25,7 @@
   ],
   "packageRules": [
     {
-      "matchPaths": [".github/workflows/assets/bootstrap/argocd-cmp-2.8/kustomization.yaml"],
-      "matchDepNames": ["argoproj/argo-cd"],
-      "matchUpdateTypes": ["major", "minor"],
-      "enabled": false
-    },
-    {
-      "matchPaths": [".github/workflows/assets/bootstrap/argocd-cmp-2.9/kustomization.yaml", "examples/installation/**"],
+      "matchPaths": [".github/workflows/assets/bootstrap/argocd-cmp-2.10/kustomization.yaml", "examples/installation/**"],
```

Comment:
Looks like this should be 2.9, as 2.10 is just below it?

Reply:
yes. It does look like that.